src/mixins/filing-template-mixin.ts

```diff
@@ -281,9 +285,13 @@ export default class FilingTemplateMixin extends DateMixin {
         this.setIncorporationAgreementStepData({
           agreementType: draftFiling.incorporationApplication.incorporationAgreement?.agreementType
         })
-        // set courtOrder attribute
-        this.setCourtOrderFileNumber(draftFiling.incorporationApplication.courtOrder?.fileNumber || '')
-        this.setHasPlanOfArrangement(draftFiling.incorporationApplication.courtOrder?.hasPlanOfArrangement || false)
+        // set court order fields
+        if (draftFiling.incorporationApplication.courtOrder?.fileNumber) {
+          this.setCourtOrderFileNumber(draftFiling.incorporationApplication.courtOrder.fileNumber)
```

Comment:
I think I prefer this style, where we don't overwrite the store initial values (and so we don't set the initial value a second time).